assets/src/admin/paired-browsing/client.js

```diff
@@ -0,0 +1,33 @@
+const { parent } = window;
+
+if ( parent.pairedBrowsingApp ) {
+	window.ampPairedBrowsingClient = true;
+	const app = parent.pairedBrowsingApp;
+
+	app.registerClientWindow( window );
+
+	document.addEventListener( 'DOMContentLoaded', () => {
```

Comment:
Could use `@wordpress/dom-ready` instead as we do already everywhere else. Example:

```js
/**
 * WordPress dependencies
 */
import domReady from '@wordpress/dom-ready';

domReady( () => {
	// ...
} );
```

It's basically the same, but also checks `document.readyState`